### PR TITLE
Update partnercentersell service

### PR DESCRIPTION
**Pull request by HarasztiaPeter**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by HarasztiaPeter** [comment]
=== RUN   TestAccIbmOnboardingResourceBrokerBasic
--- PASS: TestAccIbmOnboardingResourceBrokerBasic (65.45s)
=== RUN   TestAccIbmOnboardingResourceBrokerAllArgs
--- PASS: TestAccIbmOnboardingResourceBrokerAllArgs (68.38s)
=== RUN   TestResourceIbmOnboardingResourceBrokerBrokerEventCreatedByUserToMap
--- PASS: TestResourceIbmOnboardingResourceBrokerBrokerEventCreatedByUserToMap (0.00s)
=== RUN   TestResourceIbmOnboardingResourceBrokerBrokerEventUpdatedByUserToMap
--- PASS: TestResourceIbmOnboardingResourceBrokerBrokerEventUpdatedByUserToMap (0.00s)
=== RUN   TestResourceIbmOnboardingResourceBrokerBrokerEventDeletedByUserToMap
--- PASS: TestResourceIbmOnboardingResourceBrokerBrokerEventDeletedByUserToMap (0.00s)
PASS
ok  	command-line-arguments	135.567s

=== RUN   TestAccIbmOnboardingRegistrationBasic
--- PASS: TestAccIbmOnboardingRegistrationBasic (57.66s)
=== RUN   TestAccIbmOnboardingRegistrationAllArgs
--- PASS: TestAccIbmOnboardingRegistrationAllArgs (60.89s)
=== RUN   TestResourceIbmOnboardingRegistrationPrimaryContactToMap
--- PASS: TestResourceIbmOnboardingRegistrationPrimaryContactToMap (0.00s)
=== RUN   TestResourceIbmOnboardingRegistrationMapToPrimaryContact
--- PASS: TestResourceIbmOnboardingRegistrationMapToPrimaryContact (0.00s)
PASS
ok  	command-line-arguments	120.935s

=== RUN   TestAccIbmOnboardingProductBasic
--- PASS: TestAccIbmOnboardingProductBasic (63.84s)
=== RUN   TestAccIbmOnboardingProductAllArgs
--- PASS: TestAccIbmOnboardingProductAllArgs (62.49s)
=== RUN   TestResourceIbmOnboardingProductPrimaryContactToMap
--- PASS: TestResourceIbmOnboardingProductPrimaryContactToMap (0.00s)
=== RUN   TestResourceIbmOnboardingProductOnboardingProductSupportToMap
--- PASS: TestResourceIbmOnboardingProductOnboardingProductSupportToMap (0.00s)
=== RUN   TestResourceIbmOnboardingProductOnboardingProductSupportEscalationContactItemsToMap
--- PASS: TestResourceIbmOnboardingProductOnboardingProductSupportEscalationContactItemsToMap (0.00s)
=== RUN   TestResourceIbmOnboardingProductMapToPrimaryContact
--- PASS: TestResourceIbmOnboardingProductMapToPrimaryContact (0.00s)
=== RUN   TestResourceIbmOnboardingProductMapToOnboardingProductSupport
--- PASS: TestResourceIbmOnboardingProductMapToOnboardingProductSupport (0.00s)
=== RUN   TestResourceIbmOnboardingProductMapToOnboardingProductSupportEscalationContactItems
--- PASS: TestResourceIbmOnboardingProductMapToOnboardingProductSupportEscalationContactItems (0.00s)
PASS
ok  	command-line-arguments	127.980s

=== RUN   TestAccIbmOnboardingCatalogDeploymentBasic
--- PASS: TestAccIbmOnboardingCatalogDeploymentBasic (77.79s)
=== RUN   TestAccIbmOnboardingCatalogDeploymentAllArgs
--- PASS: TestAccIbmOnboardingCatalogDeploymentAllArgs (81.20s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogOverviewUIToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogOverviewUIToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogOverviewUITranslatedContentToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogOverviewUITranslatedContentToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentCatalogProductProviderToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentCatalogProductProviderToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogDeploymentMetadataToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogDeploymentMetadataToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIStringsToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIStringsToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIStringsContentToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIStringsContentToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentCatalogHighlightItemToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentCatalogHighlightItemToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentCatalogProductMediaItemToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentCatalogProductMediaItemToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIUrlsToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataUIUrlsToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataServiceToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataServiceToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataDeploymentToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataDeploymentToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataDeploymentBrokerToMap
--- PASS: TestResourceIbmOnboardingCatalogDeploymentGlobalCatalogMetadataDeploymentBrokerToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToCatalogProductProvider
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToCatalogProductProvider (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogOverviewUI
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogOverviewUI (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogOverviewUITranslatedContent
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogOverviewUITranslatedContent (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogDeploymentMetadata
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogDeploymentMetadata (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUI
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUI (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIStrings
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIStrings (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIStringsContent
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIStringsContent (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToCatalogHighlightItem
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToCatalogHighlightItem (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToCatalogProductMediaItem
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToCatalogProductMediaItem (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIUrls
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataUIUrls (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataService
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataService (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataDeployment
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataDeployment (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataDeploymentBroker
--- PASS: TestResourceIbmOnboardingCatalogDeploymentMapToGlobalCatalogMetadataDeploymentBroker (0.00s)
PASS
ok  	command-line-arguments	161.060s

--- PASS: TestAccIbmOnboardingCatalogPlanBasic (74.81s)
=== RUN   TestAccIbmOnboardingCatalogPlanAllArgs
--- PASS: TestAccIbmOnboardingCatalogPlanAllArgs (76.31s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogOverviewUIToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogOverviewUIToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogOverviewUITranslatedContentToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogOverviewUITranslatedContentToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanCatalogProductProviderToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanCatalogProductProviderToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogPlanMetadataToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogPlanMetadataToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIStringsToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIStringsToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIStringsContentToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIStringsContentToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanCatalogHighlightItemToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanCatalogHighlightItemToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanCatalogProductMediaItemToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanCatalogProductMediaItemToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIUrlsToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataUIUrlsToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataServiceToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataServiceToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataPricingToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogMetadataPricingToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanGlobalCatalogPlanMetadataPlanToMap
--- PASS: TestResourceIbmOnboardingCatalogPlanGlobalCatalogPlanMetadataPlanToMap (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToCatalogProductProvider
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToCatalogProductProvider (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogOverviewUI
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogOverviewUI (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogOverviewUITranslatedContent
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogOverviewUITranslatedContent (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogPlanMetadata
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogPlanMetadata (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUI
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUI (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIStrings
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIStrings (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIStringsContent
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIStringsContent (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToCatalogHighlightItem
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToCatalogHighlightItem (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToCatalogProductMediaItem
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToCatalogProductMediaItem (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIUrls
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataUIUrls (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataService
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataService (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataPricing
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogMetadataPricing (0.00s)
=== RUN   TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogPlanMetadataPlan
--- PASS: TestResourceIbmOnboardingCatalogPlanMapToGlobalCatalogPlanMetadataPlan (0.00s)
PASS
ok  	command-line-arguments	152.891s

=== RUN TestAccIbmOnboardingCatalogProductBasic
--- PASS: TestAccIbmOnboardingCatalogProductBasic (61.78s)
=== RUN TestAccIbmOnboardingCatalogProductAllArgs
--- PASS: TestAccIbmOnboardingCatalogProductAllArgs (62.10s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogOverviewUIToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogOverviewUIToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogOverviewUITranslatedContentToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogOverviewUITranslatedContentToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogProductImagesToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogProductImagesToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductCatalogProductProviderToMap
--- PASS: TestResourceIbmOnboardingCatalogProductCatalogProductProviderToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIStringsToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIStringsToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIStringsContentToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIStringsContentToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductCatalogHighlightItemToMap
--- PASS: TestResourceIbmOnboardingCatalogProductCatalogHighlightItemToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductCatalogProductMediaItemToMap
--- PASS: TestResourceIbmOnboardingCatalogProductCatalogProductMediaItemToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIUrlsToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataUIUrlsToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataServiceToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogMetadataServiceToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherPCToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherPCToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherPCSupportToMap
--- PASS: TestResourceIbmOnboardingCatalogProductGlobalCatalogProductMetadataOtherPCSupportToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductSupportEscalationToMap
--- PASS: TestResourceIbmOnboardingCatalogProductSupportEscalationToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductSupportTimeIntervalToMap
--- PASS: TestResourceIbmOnboardingCatalogProductSupportTimeIntervalToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductSupportDetailsItemToMap
--- PASS: TestResourceIbmOnboardingCatalogProductSupportDetailsItemToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductSupportDetailsItemAvailabilityToMap
--- PASS: TestResourceIbmOnboardingCatalogProductSupportDetailsItemAvailabilityToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductSupportDetailsItemAvailabilityTimeToMap
--- PASS: TestResourceIbmOnboardingCatalogProductSupportDetailsItemAvailabilityTimeToMap (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToCatalogProductProvider
--- PASS: TestResourceIbmOnboardingCatalogProductMapToCatalogProductProvider (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogOverviewUI
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogOverviewUI (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogOverviewUITranslatedContent
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogOverviewUITranslatedContent (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductImages
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductImages (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadata
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadata (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUI
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUI (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIStrings
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIStrings (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIStringsContent
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIStringsContent (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToCatalogHighlightItem
--- PASS: TestResourceIbmOnboardingCatalogProductMapToCatalogHighlightItem (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToCatalogProductMediaItem
--- PASS: TestResourceIbmOnboardingCatalogProductMapToCatalogProductMediaItem (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIUrls
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataUIUrls (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataService
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogMetadataService (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOther
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOther (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOtherPC
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOtherPC (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOtherPCSupport
--- PASS: TestResourceIbmOnboardingCatalogProductMapToGlobalCatalogProductMetadataOtherPCSupport (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToSupportEscalation
--- PASS: TestResourceIbmOnboardingCatalogProductMapToSupportEscalation (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToSupportTimeInterval
--- PASS: TestResourceIbmOnboardingCatalogProductMapToSupportTimeInterval (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItem
--- PASS: TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItem (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItemAvailability
--- PASS: TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItemAvailability (0.00s)
=== RUN TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItemAvailabilityTime
--- PASS: TestResourceIbmOnboardingCatalogProductMapToSupportDetailsItemAvailabilityTime (0.00s)
PASS

File: ibm/service/partnercentersell/resource_ibm_onboarding_resource_broker.go [hunk 1]
```diff
@@ -446,13 +470,14 @@ func resourceIbmOnboardingResourceBrokerUpdate(context context.Context, d *schem
 	updateResourceBrokerOptions := &partnercentersellv1.UpdateResourceBrokerOptions{}
 
 	updateResourceBrokerOptions.SetBrokerID(d.Id())
-	if _, ok := d.GetOk("env"); ok {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_resource_broker.go:449]: re-add this please

File: ibm/service/partnercentersell/resource_ibm_onboarding_resource_broker.go [hunk 2]
```diff
@@ -446,13 +470,14 @@ func resourceIbmOnboardingResourceBrokerUpdate(context context.Context, d *schem
...
-		updateResourceBrokerOptions.SetEnv(d.Get("env").(string))
-	}
 
 	hasChange := false
 
 	patchVals := &partnercentersellv1.BrokerPatch{}
+	if d.HasChange("env") {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_resource_broker.go:477]: remove this please

File: ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go [hunk 3]
```diff
@@ -1378,6 +1380,10 @@ func resourceIbmOnboardingIamRegistrationUpdate(context context.Context, d *sche
 			" The resource must be re-created to update this property.", "product_id")
 		return flex.DiscriminatedTerraformErrorf(nil, errMsg, "ibm_onboarding_iam_registration", "update", "product_id-forces-new").GetDiag()
 	}
+	if d.HasChange("env") {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go:1383]: remove this

Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go:1383]: remove this

File: ibm/service/partnercentersell/resource_ibm_onboarding_catalog_deployment.go [hunk 4]
```diff
@@ -573,6 +684,10 @@ func resourceIbmOnboardingCatalogDeploymentUpdate(context context.Context, d *sc
 			" The resource must be re-created to update this property.", "catalog_plan_id")
 		return flex.DiscriminatedTerraformErrorf(nil, errMsg, "ibm_onboarding_catalog_deployment", "update", "catalog_plan_id-forces-new").GetDiag()
 	}
+	if d.HasChange("env") {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_catalog_deployment.go:687]: remove this

File: ibm/service/partnercentersell/resource_ibm_onboarding_catalog_plan.go [hunk 5]
```diff
@@ -549,6 +646,10 @@ func resourceIbmOnboardingCatalogPlanUpdate(context context.Context, d *schema.R
 			" The resource must be re-created to update this property.", "catalog_product_id")
 		return flex.DiscriminatedTerraformErrorf(nil, errMsg, "ibm_onboarding_catalog_plan", "update", "catalog_product_id-forces-new").GetDiag()
 	}
+	if d.HasChange("env") {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_catalog_plan.go:649]: remove this

File: ibm/service/partnercentersell/resource_ibm_onboarding_catalog_product.go [hunk 6]
```diff
@@ -763,6 +852,10 @@ func resourceIbmOnboardingCatalogProductUpdate(context context.Context, d *schem
 			" The resource must be re-created to update this property.", "product_id")
 		return flex.DiscriminatedTerraformErrorf(nil, errMsg, "ibm_onboarding_catalog_product", "update", "product_id-forces-new").GetDiag()
 	}
+	if d.HasChange("env") {
```
Comment (balazs-marjan) [inline ibm/service/partnercentersell/resource_ibm_onboarding_catalog_product.go:855]: remove this

File: ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go [hunk 7]
```diff
@@ -46,7 +46,7 @@ func ResourceIbmOnboardingIamRegistration() *schema.Resource {
 			},
 			"name": &schema.Schema{
 				Type:         schema.TypeString,
-				Optional:     true,
+				Required:     true,
```
Comment (hkantare) [inline ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go:49]: Changing from Optional to Required are kind of breaking changes for user's . IF its required share me notes section which I can included in release notes

**Comment by hkantare** [comment]
Remaining looks fine

File: ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go (hunk 7 above)
Comment (HarasztiaPeter) [reply ibm/service/partnercentersell/resource_ibm_onboarding_iam_registration.go:49]: All of our documents contains the following notice:
**Note - Intended for internal use only. This resource is strictly experimental and subject to change without notice.**

**Review by balazs-marjan — APPROVED** [review]

**Review by hkantare — APPROVED** [review]

**Pull request merged**